src/controller/content/dto/content.dto.ts

```diff
@@ -272,6 +272,17 @@ export class  quizBatchDto {
 
 }
 
+export class CreateQuizzesDto {
+  @ApiProperty({
+    type: [quizBatchDto],
+    required: true,
+  })
+  @IsArray()
+  @ValidateNested({ each: true })
+  @Type(() => quizBatchDto)
+  quizzes: quizBatchDto[];
```

Comment:
If you are mapping the quizBatchDto to the quizzes, ensure to validate the received payload according to the required  attributes in the array of objects. @sukanyamohanty143